soccer/gameplay/evaluation/touchpass_positioning.py

```diff
@@ -0,0 +1,100 @@
+import robocup
+import constants
+import main
+import evaluation.passing
+
+## The Touchpass Positioner finds the best location within a rectangle
+#
+# Example usage:
+# tpos = TouchpassPositioner()
+# best_pt = tpos.evaluate(ball.pos, robot2.pos)
+class TouchpassPositioner:
+
+    def __init__(self):
+        self.debug = False
+
+    # Defaults to False
+    # if True, uses the system state drawing methods to draw Windows
+    @property
+    def debug(self):
+        return self._debug
+    @debug.setter
+    def debug(self, value):
+        self._debug = value
+
+    ## Returns a robocup.Rect object that is the default location to be evaluated
+    #
+    # Takes a current ball position/initial kick position (robocup.Point)
+    def generate_default_rectangle(self, kick_point):
```

Comment:
This could probably be a class method since it doesn't depend on any instance state.